spaces/ZZZ.finance/index.json

```
@@ -0,0 +1,34 @@
+{
+  "token": ""0xc75f15ada581219c95485c578e124df3985e4ce0",
```

You have a syntax error here.
```suggestion
  "token": "0xc75f15ada581219c95485c578e124df3985e4ce0",
```

---

Your folder name should contains only lowercase and no ponctuation. You should rename `spaces/ZZZ.finance/index.json` to something like that `spaces/zzz-finance/index.json`.